README.md

```diff
@@ -39,15 +39,18 @@
 By using this software you agree to the terms in COPYING
 
 ## Installation:
- - Install TensorFlow(>1.1), otherwise TensorMol is self-contained.
+ - Install TensorFlow(>1.1) as well as scipy, otherwise TensorMol is self-contained.
  - Works on OSX, Ubuntu, and Windows subsystem for Linux:
 ```
-git clone https://github.com/jparkhill/TensorMol.git
+git clone git@github.com:Exabyte-io/TensorMol.git 
+git checkout dev
 cd TensorMol
+pip install -r requirements.txt
 # If you are using python2x
-sudo pip install -e .
+pip install -e .
 # If you are using python3x
-sudo pip3 install -e .
+pip3 install -e .
+cd samples
```

Comment:
We should add a directive for creating a virtual environment instead of using sudo, and isolate the python2 and 3 cases entirely

Reply:
Updated readme to avoid the use of sudo.